lib/clientv3/globalnetworkpolicy.go

```diff
@@ -216,6 +220,10 @@ func convertPolicyNameFromStorage(name string) string {
 	if strings.HasPrefix(name, "knp.") {
 		return name
 	}
+	// Similarly for "ossg."
+	if strings.HasPrefix(name, "ossg.") {
```

Comment:
This function is in the _global_ network policies file but I thought we're writing namespaced network policies?

Reply:
Looks like the functions are shared and used in both